### PR TITLE
docs(guides): use staging server for development (#912)

### PR DESCRIPTION
**Pull request by marswong**

### CODE REVIEW

File: guides/FRONTEND.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ yarn
 To connect to production servers, you'll need to create a file `.env.local` under `packages/web` and write this inside:
```
Comment (polimyl) [inline guides/FRONTEND.md:22]: you changed `NEXT_PUBLIC_GRAPHQL_URL` to use the staging backend, while this line and `.env.sample` still refer to the production backend (api.metagame.wtf). all of them should refer to the same backend instance.

**Review by polimyl — CHANGES_REQUESTED** [review]

File: guides/FRONTEND.md (hunk 1 above)
Comment (marswong) [reply guides/FRONTEND.md:22]: good job. updated :)

Comment (polimyl) [reply guides/FRONTEND.md:22]: line 22 in `FRONTEND.md` still refers to "production servers" :P

Comment (marswong) [reply guides/FRONTEND.md]: updated

**Review by polimyl — APPROVED** [review]
lgtm

**Pull request merged**